website/azurerm.erb

```diff
@@ -0,0 +1,3169 @@
+
```

Comment:
We've finally moved away from this format so we don't have to maintain the sidebar anymore! We can save some headache and remove this file